awx/main/tasks.py

```diff
@@ -660,6 +660,25 @@ def update_host_smart_inventory_memberships():
         smart_inventory.update_computed_fields()
 
 
+@task()
+def migrate_legacy_event_data(tblname, total_rows):
```

Comment:
I should probably remove this `total_rows` parameter and just calculate it by hand here, so that you don't have to know it beforehand to `.apply_async()` this task manually.